test/L2/L2AirdropV2.t.sol

```diff
@@ -0,0 +1,594 @@
+        assertEq(l2AirdropV2.claimedStakingTier2(aliceLSKAddress), false);
+
+        // claim airdrop for alice (only staking tier 2 condition is satisfied)
+        uint256 aliceBalanceBefore = l2LiskToken.balanceOf(alice);
+        bytes32[] memory merkleProof = new bytes32[](1);
+        merkleProof[0] = bytes32(0xf0df3dcda05b4fbd9c655cde3d5ceb211e019e72ec816e127a59e7195f2cd7f5);
+        l2AirdropV2.claimAirdrop(aliceLSKAddress, 80 * 10 ** 18, merkleProof);
+        assertEq(l2LiskToken.balanceOf(alice), aliceBalanceBefore + 40 * 10 ** 18); // 4 L2LiskToken airdrop
+
+        // check that alice has claimed airdrop for staking tier 2 condition
+        assertEq(l2AirdropV2.claimedStakingTier2(aliceLSKAddress), true);
+    }
+
+    function test_ClaimAirdrop_StakingTier2() public {
+        // first alice will claim airdrop for staking tier 1 condition that only staking tier 2 condition will be left
+        aliceClaimAirdropForStakingTier1();
+
+        // check that alice can claim airdrop for staking tier 2 condition
+        aliceClaimAirdropForStakingTier2();
+    }
+
+    function test_ClaimAirdrop_StakingTier1_StakingTier2() public {
+        // alice satisfies staking tier 1 condition
+        aliceSatifiesStakingTier1();
+
+        // alice satisfies staking tier 2 condition
+        aliceSatifiesStakingTier2();
+
+        uint256 aliceBalanceBefore = l2LiskToken.balanceOf(alice);
+        bytes32[] memory merkleProof = new bytes32[](1);
+        merkleProof[0] = bytes32(0xf0df3dcda05b4fbd9c655cde3d5ceb211e019e72ec816e127a59e7195f2cd7f5);
+        l2AirdropV2.claimAirdrop(aliceLSKAddress, 80 * 10 ** 18, merkleProof);
+        assertEq(l2LiskToken.balanceOf(alice), aliceBalanceBefore + 80 * 10 ** 18); // 80 L2LiskToken airdrop
+
+        // check airdrop claim status for alice
+        assertEq(l2AirdropV2.claimedStakingTier1(bytes20(alice)), true);
+        assertEq(l2AirdropV2.claimedStakingTier2(bytes20(alice)), true);
+    }
+
+    function test_ClaimAirdrop_FullAirdrop() public {
+        // alice satisfies staking tier 1 condition
+        aliceSatifiesStakingTier1();
+
+        // alice satisfies staking tier 2 condition
+        aliceSatifiesStakingTier2();
+
+        uint256 aliceBalanceBefore = l2LiskToken.balanceOf(alice);
+        bytes32[] memory merkleProof = new bytes32[](1);
+        merkleProof[0] = bytes32(0xf0df3dcda05b4fbd9c655cde3d5ceb211e019e72ec816e127a59e7195f2cd7f5);
+
+        // check that the AirdropClaimed event is emitted for all conditions
+        vm.expectEmit(true, true, true, true);
+        emit L2AirdropV2.AirdropClaimed(
+            aliceLSKAddress, 80 * 10 ** 18, alice, l2AirdropV2.STAKING_TIER_1_BIT() | l2AirdropV2.STAKING_TIER_2_BIT()
+        );
+
+        l2AirdropV2.claimAirdrop(aliceLSKAddress, 80 * 10 ** 18, merkleProof);
+        assertEq(l2LiskToken.balanceOf(alice), aliceBalanceBefore + 80 * 10 ** 18); // 16 L2LiskToken airdrop
```

Comment:
either remove the comment or change 16 to 80 if you think it makes sense (for me, the meaning is unclear)